### ISSUE TITLE
ERROR:  relation "dbpatch" does not exist at character 42

### ISSUE THREAD

**Issue by thommc** [openmicroscopy/omero-test-daemon-c7-docker#4]
Hi.  Thanks for providing these images.

We can't seem to invoke this docker image without the following error.  I've cleared the images, pulled fresh ones repeatedly, updated Docker to 1.12 from their repos (in RHEL7) and tried holding back the Postgre version to 9.5 to no avail.

Not sure quite what's going wrong here.  Grateful for your input.

```
Running with gitlab-ci-multi-runner 1.8.1 (a2efdd4)
Using Shell executor...
Running on jervis-docker...
Fetching changes...
HEAD is now at 72c7dc5 amend test container name in exit code query
Checking out 72c7dc5a as master...
$ docker info
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 0
Server Version: 1.10.3
Storage Driver: devicemapper
 Pool Name: docker-253:0-26208-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 3.025 GB
 Data Space Total: 107.4 GB
 Data Space Available: 10.12 GB
 Metadata Space Used: 4.964 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.143 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.135-RHEL7 (2016-09-28)
Execution Driver: native-0.2
Logging Driver: journald
Plugins:
 Volume: local
 Network: host bridge null
 Authorization: rhel-push-plugin
Kernel Version: 3.10.0-514.2.2.el7.x86_64
Operating System: Red Hat Enterprise Linux
OSType: linux
Architecture: x86_64
Number of Docker Hooks: 2
CPUs: 2
Total Memory: 3.702 GiB
Name: jervis-docker
ID: VDDE:BKJM:7XUS:B3ZX:TS34:2HN3:IY7F:WVAO:DRBG:QDJY:ZIHP:H3TX
Registries: registry.access.redhat.com (secure), docker.io (secure)
$ docker-compose -f docker-compose.yml up --build --abort-on-container-exit
Pulling postgres (postgres:latest)...
Trying to pull repository registry.access.redhat.com/postgres ...
unknown: Not Found
Trying to pull repository docker.io/library/postgres ...
latest: Pulling from docker.io/library/postgres
Digest: sha256:855b55b060bfa63cb39bfee0fe4ec6f40037e9f85f8a7eec24bc8e59ec1a8345
Status: Downloaded newer image for docker.io/postgres:latest
Pulling omeroserver (openmicroscopy/omero-test-daemon-c7:0.2.0)...
Trying to pull repository registry.access.redhat.com/openmicroscopy/omero-test-daemon-c7 ...
unknown: Not Found
Trying to pull repository docker.io/openmicroscopy/omero-test-daemon-c7 ...
0.2.0: Pulling from docker.io/openmicroscopy/omero-test-daemon-c7
Digest: sha256:b057734fecdca07295244808dd605e70e9ecf5b3c962b128c2f4731ac9a6eb79
Status: Downloaded newer image for docker.io/openmicroscopy/omero-test-daemon-c7:0.2.0
Creating rdm_postgres_1
Creating rdm_omeroserver_1
Creating rdm_omeropy_1
Attaching to rdm_postgres_1, rdm_omeroserver_1, rdm_omeropy_1
postgres_1     | The files belonging to this database system will be owned by user "postgres".
postgres_1     | This user must also own the server process.
postgres_1     |
postgres_1     | The database cluster will be initialized with locale "en_US.utf8".
postgres_1     | The default database encoding has accordingly been set to "UTF8".
postgres_1     | The default text search configuration will be set to "english".
postgres_1     |
postgres_1     | Data page checksums are disabled.
postgres_1     |
postgres_1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
postgres_1     | creating subdirectories ... ok
postgres_1     | selecting default max_connections ... 100
postgres_1     | selecting default shared_buffers ... 128MB
postgres_1     | selecting dynamic shared memory implementation ... posix
postgres_1     | creating configuration files ... ok
postgres_1     | running bootstrap script ... ok
omeroserver_1  | + OMEROBUILD=
omeroserver_1  | + COMPONENT=server
omeroserver_1  | + RUNSCRIPT=server
omeroserver_1  | + OMEROPATH=/home/omero/OMERO.server
omeroserver_1  | + sudo pip install -r /tmp/install/requirements.txt
omeroserver_1  | Collecting omego (from -r /tmp/install/requirements.txt (line 7))
omeroserver_1  |   Downloading omego-0.4.2.tar.gz
omeroserver_1  | Collecting zeroc-ice<3.7,>3.5 (from -r /tmp/install/requirements.txt (line 8))
omeroserver_1  |   Downloading zeroc-ice-3.6.3.tar.gz (1.1MB)
postgres_1     | performing post-bootstrap initialization ... ok
omeropy_1      | + OMEROBUILD=
omeropy_1      | + COMPONENT=py
omeropy_1      | + RUNSCRIPT=server
omeropy_1      | + OMEROPATH=/home/omero/OMERO.py
omeropy_1      | + sudo pip install -r /tmp/install/requirements.txt
postgres_1     | syncing data to disk ...
postgres_1     | WARNING: enabling "trust" authentication for local connections
postgres_1     | You can change this by editing pg_hba.conf or using the option -A, or
postgres_1     | --auth-local and --auth-host, the next time you run initdb.
postgres_1     | ok
postgres_1     |
postgres_1     | Success. You can now start the database server using:
postgres_1     |
postgres_1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
postgres_1     |
postgres_1     | ****************************************************
postgres_1     | WARNING: No password has been set for the database.
postgres_1     |          This will allow anyone with access to the
postgres_1     |          Postgres port to access your database. In
postgres_1     |          Docker's default configuration, this is
postgres_1     |          effectively any other container on the same
postgres_1     |          system.
postgres_1     |
postgres_1     |          Use "-e POSTGRES_PASSWORD=password" to set
postgres_1     |          it in "docker run".
postgres_1     | ****************************************************
postgres_1     | waiting for server to start....LOG:  database system was shut down at 2016-12-20 05:47:42 UTC
postgres_1     | LOG:  MultiXact member wraparound protections are now enabled
postgres_1     | LOG:  database system is ready to accept connections
postgres_1     | LOG:  autovacuum launcher started
omeroserver_1  | Collecting yaclifw>=0.1.1 (from omego->-r /tmp/install/requirements.txt (line 7))
omeroserver_1  |   Downloading yaclifw-0.1.2.tar.gz
omeropy_1      | Collecting omego (from -r /tmp/install/requirements.txt (line 7))
omeropy_1      |   Downloading omego-0.4.2.tar.gz
omeroserver_1  | Collecting argparse (from yaclifw>=0.1.1->omego->-r /tmp/install/requirements.txt (line 7))
omeroserver_1  |   Downloading argparse-1.4.0-py2.py3-none-any.whl
omeroserver_1  | Installing collected packages: argparse, yaclifw, omego, zeroc-ice
omeroserver_1  |   Running setup.py install for yaclifw: started
omeropy_1      | Collecting zeroc-ice<3.7,>3.5 (from -r /tmp/install/requirements.txt (line 8))
omeropy_1      |   Downloading zeroc-ice-3.6.3.tar.gz (1.1MB)
postgres_1     |  done
postgres_1     | server started
postgres_1     | ALTER ROLE
postgres_1     |
postgres_1     |
postgres_1     | /docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
postgres_1     |
postgres_1     | LOG:  received fast shutdown request
postgres_1     | LOG:  aborting any active transactions
postgres_1     | waiting for server to shut down....LOG:  autovacuum launcher shutting down
postgres_1     | LOG:  shutting down
omeroserver_1  |     Running setup.py install for yaclifw: finished with status 'done'
omeroserver_1  |   Running setup.py install for omego: started
postgres_1     | LOG:  database system is shut down
omeroserver_1  |     Running setup.py install for omego: finished with status 'done'
omeroserver_1  |   Running setup.py install for zeroc-ice: started
omeropy_1      | Collecting yaclifw>=0.1.1 (from omego->-r /tmp/install/requirements.txt (line 7))
omeropy_1      |   Downloading yaclifw-0.1.2.tar.gz
postgres_1     |  done
postgres_1     | server stopped
postgres_1     |
postgres_1     | PostgreSQL init process complete; ready for start up.
postgres_1     |
omeropy_1      | Collecting argparse (from yaclifw>=0.1.1->omego->-r /tmp/install/requirements.txt (line 7))
postgres_1     | LOG:  database system was shut down at 2016-12-20 05:47:43 UTC
postgres_1     | LOG:  MultiXact member wraparound protections are now enabled
postgres_1     | LOG:  database system is ready to accept connections
postgres_1     | LOG:  autovacuum launcher started
omeropy_1      |   Downloading argparse-1.4.0-py2.py3-none-any.whl
omeropy_1      | Installing collected packages: argparse, yaclifw, omego, zeroc-ice
omeropy_1      |   Running setup.py install for yaclifw: started
omeropy_1      |     Running setup.py install for yaclifw: finished with status 'done'
omeropy_1      |   Running setup.py install for omego: started
omeropy_1      |     Running setup.py install for omego: finished with status 'done'
omeropy_1      |   Running setup.py install for zeroc-ice: started
omeroserver_1  |     Running setup.py install for zeroc-ice: still running...
omeropy_1      |     Running setup.py install for zeroc-ice: still running...
omeroserver_1  |     Running setup.py install for zeroc-ice: still running...
omeropy_1      |     Running setup.py install for zeroc-ice: still running...
omeroserver_1  |     Running setup.py install for zeroc-ice: still running...
omeropy_1      |     Running setup.py install for zeroc-ice: still running...
omeroserver_1  |     Running setup.py install for zeroc-ice: still running...
omeropy_1      |     Running setup.py install for zeroc-ice: still running...
omeroserver_1  |     Running setup.py install for zeroc-ice: still running...
omeropy_1      |     Running setup.py install for zeroc-ice: still running...
omeropy_1      |     Running setup.py install for zeroc-ice: still running...
omeroserver_1  |     Running setup.py install for zeroc-ice: still running...
omeroserver_1  |     Running setup.py install for zeroc-ice: finished with status 'done'
omeropy_1      |     Running setup.py install for zeroc-ice: finished with status 'done'
omeroserver_1  | Successfully installed argparse-1.4.0 omego-0.4.2 yaclifw-0.1.2 zeroc-ice-3.6.3
omeropy_1      | Successfully installed argparse-1.4.0 omego-0.4.2 yaclifw-0.1.2 zeroc-ice-3.6.3
omeroserver_1  | You are using pip version 8.1.2, however version 9.0.1 is available.
omeroserver_1  | You should consider upgrading via the 'pip install --upgrade pip' command.
omeropy_1      | You are using pip version 8.1.2, however version 9.0.1 is available.
omeropy_1      | You should consider upgrading via the 'pip install --upgrade pip' command.
omeroserver_1  | + echo 'Downloading  OMERO.server...'
omeroserver_1  | + ARGS=
omeroserver_1  | + '[' -n '' ']'
omeroserver_1  | + cd /tmp
omeroserver_1  | + omego download --ice 3.6 server
omeroserver_1  | Downloading  OMERO.server...
omeropy_1      | + echo 'Downloading  OMERO.py...'
omeropy_1      | Downloading  OMERO.py...
omeropy_1      | + ARGS=
omeropy_1      | + '[' -n '' ']'
omeropy_1      | + cd /tmp
omeropy_1      | + omego download --ice 3.6 py
omeropy_1      | 2016-12-20 05:55:08,428 [omego.artifa] INFO  Checking http://downloads.openmicroscopy.org/omero/5.2.7/./artifacts/OMERO.py-5.2.7-ice36-b40.zip
omeropy_1      | 2016-12-20 05:55:08,428 [omego.fileut] INFO  Downloading http://downloads.openmicroscopy.org/omero/5.2.7/./artifacts/OMERO.py-5.2.7-ice36-b40.zip
omeroserver_1  | 2016-12-20 05:55:08,490 [omego.artifa] INFO  Checking http://downloads.openmicroscopy.org/omero/5.2.7/./artifacts/OMERO.server-5.2.7-ice36-b40.zip
omeroserver_1  | 2016-12-20 05:55:08,491 [omego.fileut] INFO  Downloading http://downloads.openmicroscopy.org/omero/5.2.7/./artifacts/OMERO.server-5.2.7-ice36-b40.zip
omeropy_1      | 2016-12-20 05:55:13,070 [omego.artifa] INFO  Unzipping OMERO.py-5.2.7-ice36-b40.zip
omeropy_1      | ++ ls /tmp/OMERO.py-5.2.7-ice36-b40.zip
omeropy_1      | + omegozip=/tmp/OMERO.py-5.2.7-ice36-b40.zip
omeropy_1      | + omerodist=/tmp/OMERO.py-5.2.7-ice36-b40
omeropy_1      | + rm -f /tmp/OMERO.py-5.2.7-ice36-b40.zip
omeropy_1      | + mv /tmp/OMERO.py-5.2.7-ice36-b40 /home/omero/OMERO.py
omeropy_1      | + /tmp/install/server.sh /home/omero/OmeroPy/bin/omero config get
omeropy_1      | + COMPONENT=py
omeropy_1      | + OMEROPATH=/home/omero/OMERO.py
omeropy_1      | + echo 'Waiting on Postgres...'
omeropy_1      | + psql -h postgres -p 5432 -U postgres -c '\l'
omeropy_1      | Waiting on Postgres...
omeropy_1      |                                  List of databases
omeropy_1      |    Name    |  Owner   | Encoding |  Collate   |   Ctype    |   Access privileges  
omeropy_1      | -----------+----------+----------+------------+------------+-----------------------
omeropy_1      |  postgres  | postgres | UTF8     | en_US.utf8 | en_US.utf8 |
omeropy_1      |  template0 | postgres | UTF8     | en_US.utf8 | en_US.utf8 | =c/postgres          +
omeropy_1      |            |          |          |            |            | postgres=CTc/postgres
omeropy_1      |  template1 | postgres | UTF8     | en_US.utf8 | en_US.utf8 | =c/postgres          +
omeropy_1      |            |          |          |            |            | postgres=CTc/postgres
omeropy_1      | (3 rows)
omeropy_1      |
omeropy_1      | + echo 'postgres now accepts connections, creating database omero...'
omeropy_1      | postgres now accepts connections, creating database omero...
omeropy_1      | + echo 'Loading config...'
omeropy_1      | + /home/omero/OMERO.py/bin/omero config set omero.db.host postgres
omeropy_1      | Loading config...
omeropy_1      | + /home/omero/OMERO.py/bin/omero config set omero.db.port 5432
omeropy_1      | + /home/omero/OMERO.py/bin/omero config set omero.db.user postgres
omeropy_1      | + /home/omero/OMERO.py/bin/omero config set omero.db.name omero
omeropy_1      | + '[' false = true ']'
omeropy_1      | + psql -h postgres -p 5432 -U postgres -lqt
omeropy_1      | + grep -qw omero
omeropy_1      | + cut -d '|' -f 1
omeropy_1      | + createdb -h postgres -p 5432 -U postgres omero
omeropy_1      | + echo 'INFO: omero was created.'
omeropy_1      | + omego db upgrade --serverdir /home/omero/OMERO.py --dbname omero
omeropy_1      | INFO: omero was created.
omeropy_1      | 2016-12-20 05:55:22,632 [    omego.db] INFO  DbAdmin: DbAdmin /home/omero/OMERO.py ...
omeropy_1      | 2016-12-20 05:55:22,633 [    omego.db] INFO  Failed to lookup parameter omero.db.pass, using omero
omeropy_1      | 2016-12-20 05:55:22,633 [omego.extern] INFO  Executing [custom environment]: psql -d omero -h postgres -U postgres -w -A -t --version
omeropy_1      | 2016-12-20 05:55:22,645 [omego.extern] INFO  Completed [0.012 s]
omeropy_1      | 2016-12-20 05:55:22,646 [    omego.db] INFO  psql version: psql (PostgreSQL) 9.5.5
omeropy_1      |
omeropy_1      | 2016-12-20 05:55:22,647 [    omego.db] INFO  Failed to lookup parameter omero.db.pass, using omero
omeropy_1      | 2016-12-20 05:55:22,647 [omego.extern] INFO  Executing [custom environment]: psql -d omero -h postgres -U postgres -w -A -t -c \conninfo
omeropy_1      | 2016-12-20 05:55:22,671 [omego.extern] INFO  Completed [0.024 s]
omeropy_1      | 2016-12-20 05:55:22,672 [    omego.db] INFO  Failed to lookup parameter omero.db.pass, using omero
omeropy_1      | 2016-12-20 05:55:22,672 [omego.extern] INFO  Executing [custom environment]: psql -d omero -h postgres -U postgres -w -A -t -c SELECT currentversion, currentpatch FROM dbpatch ORDER BY id DESC LIMIT 1
postgres_1     | ERROR:  relation "dbpatch" does not exist at character 42
postgres_1     | STATEMENT:  SELECT currentversion, currentpatch FROM dbpatch ORDER BY id DESC LIMIT 1
omeropy_1      | 2016-12-20 05:55:22,692 [omego.extern] ERROR Failed [0.020 s]
omeropy_1      | Traceback (most recent call last):
omeropy_1      |   File "/usr/bin/omego", line 9, in <module>
omeropy_1      |     load_entry_point('omego==0.4.2', 'console_scripts', 'omego')()
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/main.py", line 52, in entry_point
omeropy_1      |     (Version.NAME, Version)])
omeropy_1      |   File "/usr/lib/python2.7/site-packages/yaclifw/framework.py", line 197, in main
omeropy_1      |     ns.func(ns)
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/db.py", line 309, in __call__
omeropy_1      |     DbAdmin(d, args.dbcommand, args, ext)
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/db.py", line 78, in __init__
omeropy_1      |     getattr(self, command)()
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/db.py", line 157, in upgrade
omeropy_1      |     currentsqlv = '%s__%s' % self.get_current_db_version()
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/db.py", line 174, in get_current_db_version
omeropy_1      |     result = self.psql('-c', q)
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/db.py", line 240, in psql
omeropy_1      |     stdout, stderr = External.run('psql', args, capturestd=True, env=env)
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/external.py", line 223, in run
omeropy_1      |     "Non-zero return code", exe, args, r, stdout, stderr)
omeropy_1      | omego.external.RunException: Non-zero return code
omeropy_1      | command: psql -d omero -h postgres -U postgres -w -A -t -c SELECT currentversion, currentpatch FROM dbpatch ORDER BY id DESC LIMIT 1
omeropy_1      | return code: 1
omeropy_1      | stdout:
omeropy_1      | stderr: ERROR:  relation "dbpatch" does not exist
omeropy_1      | LINE 1: SELECT currentversion, currentpatch FROM dbpatch ORDER BY id...
omeropy_1      |                                                  ^
omeropy_1      |
omeropy_1      | + omego db init --serverdir /home/omero/OMERO.py --dbname omero
omeropy_1      | 2016-12-20 05:55:23,748 [    omego.db] INFO  DbAdmin: DbAdmin /home/omero/OMERO.py ...
omeropy_1      | 2016-12-20 05:55:23,750 [    omego.db] INFO  Failed to lookup parameter omero.db.pass, using omero
omeropy_1      | 2016-12-20 05:55:23,750 [omego.extern] INFO  Executing [custom environment]: psql -d omero -h postgres -U postgres -w -A -t --version
omeropy_1      | 2016-12-20 05:55:23,763 [omego.extern] INFO  Completed [0.013 s]
omeropy_1      | 2016-12-20 05:55:23,763 [    omego.db] INFO  psql version: psql (PostgreSQL) 9.5.5
omeropy_1      |
omeropy_1      | 2016-12-20 05:55:23,764 [    omego.db] INFO  Failed to lookup parameter omero.db.pass, using omero
omeropy_1      | 2016-12-20 05:55:23,764 [omego.extern] INFO  Executing [custom environment]: psql -d omero -h postgres -U postgres -w -A -t -c \conninfo
omeropy_1      | 2016-12-20 05:55:23,784 [omego.extern] INFO  Completed [0.020 s]
omeropy_1      | 2016-12-20 05:55:23,784 [    omego.db] INFO  Creating SQL: omero-20161220-055523-784727.sql
omeropy_1      | 2016-12-20 05:55:23,784 [omego.extern] INFO  Invoking CLI [current environment]: db script -f omero-20161220-055523-784727.sql   omero
omeropy_1      | WARNING: Positional arguments are deprecated
omeropy_1      | Using OMERO5.2 for version
omeropy_1      | Using 0 for patch
omeropy_1      | Using password from commandline
omeropy_1      | Invalid model-*.jar state:
omeropy_1      | Traceback (most recent call last):
omeropy_1      |   File "/usr/bin/omego", line 9, in <module>
omeropy_1      |     load_entry_point('omego==0.4.2', 'console_scripts', 'omego')()
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/main.py", line 52, in entry_point
omeropy_1      |     (Version.NAME, Version)])
omeropy_1      |   File "/usr/lib/python2.7/site-packages/yaclifw/framework.py", line 197, in main
omeropy_1      |     ns.func(ns)
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/db.py", line 309, in __call__
omeropy_1      |     DbAdmin(d, args.dbcommand, args, ext)
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/db.py", line 78, in __init__
omeropy_1      |     getattr(self, command)()
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/db.py", line 98, in init
omeropy_1      |     self.args.rootpass])
omeropy_1      |   File "/usr/lib/python2.7/site-packages/omego/external.py", line 166, in omero_cli
omeropy_1      |     self.cli.invoke(command, strict=True)
omeropy_1      |   File "/home/omero/OMERO.py/lib/python/omero/cli.py", line 955, in invoke
omeropy_1      |     self.assertRC()
omeropy_1      |   File "/home/omero/OMERO.py/lib/python/omero/cli.py", line 944, in assertRC
omeropy_1      |     raise NonZeroReturnCode(self.rv, "assert failed")
omeropy_1      | omero.cli.NonZeroReturnCode: assert failed
rdm_omeropy_1 exited with code 1
Stopping rdm_omeroserver_1 ...
 
Stopping rdm_postgres_1 ...
 
 
Stopping rdm_omeroserver_1 ... done
 
Stopping rdm_postgres_1 ... done
Aborting on container exit...
$ exitcode=$(docker inspect -f '{{.State.ExitCode}}' rdm_omeropy_1)
$ docker-compose -f docker-compose.yml rm -fv
Removing rdm_omeropy_1 ...
 
Removing rdm_omeroserver_1 ...
 
Removing rdm_postgres_1 ...
 
 
Removing rdm_omeropy_1 ... done
 
Removing rdm_omeroserver_1 ... done
 
Removing rdm_postgres_1 ... done
Going to remove rdm_omeropy_1, rdm_omeroserver_1, rdm_postgres_1
$ exit $exitcode
Running after script...
$ docker rm `docker ps --no-trunc -aq`
docker: "rm" requires a minimum of 1 argument.
See '/usr/bin/docker-current rm --help'.
 
Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]
 
Remove one or more containers
ERROR: Build failed: exit status 1
```

**Comment by aleksandra-tarkowska** [comment]
Could you show me your compose file?

**Comment by aleksandra-tarkowska** [comment]
did you follow example in https://github.com/openmicroscopy/omero-test-daemon-c7-docker/blob/master/test/docker-compose.yml

default params should give you output like https://travis-ci.org/aleksandra-tarkowska/omero-test-daemon-c7-docker/jobs/185724773

You would need to show me exactly what did you changed

**Comment by joshmoore** [comment]
cF. http://pastebin.com/5MC8rVLu

**Comment by aleksandra-tarkowska** [comment]
I have made it clear in readme now in https://github.com/openmicroscopy/omero-test-daemon-c7-docker/pull/5/

**Comment by thommc** [comment]
Right on, got this running now.  Thanks @joshmoore , @sbesson and especially @aleksandra-tarkowska for all your help - love ya work.  Have a great break!

**Comment by joshmoore** [comment]
:+1:
